yt_dlp/extractor/dw.py

```diff
@@ -1,111 +1,164 @@
-            'description': 'md5:bc9ca6e4e063361e21c920c53af12405',
-            'upload_date': '20160311',
+            'title': '30 years since the fall of the Berlin Wall: What happened to the euphoria?',
+            'categories': ['Politics'],
+            'description': 'md5:ed0686725ef2d8fa5230aa8171b6476b',
+            'duration': 1561,
+            'release_date': '20191107',
+            'release_timestamp': 1573140547,
+            'thumbnail': r're:https?://static\.dw\.com/image/.+\.webp',
         },
     }, {
-        # DW documentaries, only last for one or two weeks
-        'url': 'http://www.dw.com/en/documentaries-welcome-to-the-90s-2016-05-21/e-19220158-9798',
-        'md5': '56b6214ef463bfb9a3b71aeb886f3cf1',
+        'url': 'https://www.dw.com/en/10-years-since-higgs-boson-whats-next/a-62353123',
         'info_dict': {
-            'id': '19274438',
-            'ext': 'mp4',
-            'title': 'Welcome to the 90s – Hip Hop',
-            'description': 'Welcome to the 90s - The Golden Decade of Hip Hop',
-            'upload_date': '20160521',
+            'id': '62353123',
+            'title': '10 years since Higgs boson. What\'s next?',
+        },
+        'playlist_count': 1,
+    }, {
+        'url': 'https://www.dw.com/am/31-%E1%8B%93%E1%88%98%E1%89%B5-%E1%8A%A0%E1%8C%88%E1%88%8D%E1%8C%8D%E1%88%88%E1%8B%8D-%E1%8A%A8100-%E1%8B%B6%E1%88%8B%E1%88%AD-%E1%8B%AB%E1%8A%90%E1%88%B0-%E1%88%B6%E1%88%9B%E1%88%8A%E1%88%8B%E1%8A%95%E1%8B%B5-4%E1%88%BA%E1%88%85-%E1%8B%B6%E1%88%8B%E1%88%AD-%E1%8B%A8%E1%8B%88%E1%88%AD-%E1%8A%AD%E1%8D%8D%E1%8B%AB-%E1%8B%B6-%E1%88%AD-%E1%8D%8D%E1%88%B5%E1%88%83/a-72565331',
+        'info_dict': {
+            'id': '72565331',
+            'title': 'md5:b01a937d9023dc9d6e96f0c30e86816e',
+        },
+        'playlist_count': 1,
+    }, {
+        'url': 'https://amp.dw.com/am/%E1%88%81%E1%88%88%E1%89%B5-%E1%8B%A8%E1%88%98%E1%8A%95%E1%8C%8D%E1%88%B5%E1%89%B5-%E1%8D%8B%E1%89%A5%E1%88%AA%E1%8A%AB%E1%8B%8E%E1%89%BD-%E1%8B%88%E1%8B%B0-%E1%8C%A5%E1%88%A8%E1%89%B5-%E1%88%8A%E1%8B%9B%E1%8B%88%E1%88%A9-%E1%8A%90%E1%8B%8D/a-37166407',
+        'info_dict': {
+            'id': '37166407',
+            'title': 'md5:56fc6855d20de908fc5bd6cff226ddb4',
+        },
+        'playlist_count': 1,
+    }, {
+        'url': 'https://www.dw.com/en/0/live-72516011',
+        'info_dict': {
+            'id': '72516011',
+            'title': 'Germany updates: Afghan family sues government — report',
+        },
+        'playlist_count': 5,
+    }, {
+        'url': 'https://www.dw.com/en/choices/program-68776240',
+        'info_dict': {
+            'id': '68776240',
+            'title': 'CHOICES',
         },
-        'skip': 'Video removed',
+        'playlist_count': 13,
     }]
 
+    def _entries(self, url, graph_api, media_type):
+        path = self._ENTRIES_PATH_MAP[media_type]
+        for dct in traverse_obj(graph_api, (
+            *variadic(path), lambda _, v: v['namedUrl'] not in url,
```

Comment:
```suggestion
        for dct in traverse_obj(graph_api, (
            *variadic(self._ENTRIES_PATH_MAP[media_type]), lambda _, v: v['namedUrl'] not in url,
```
alternatively, we could just make it so all of the `_ENTRIES_PATH_MAP` values are tuples instead of using variadic

Reply:
> alternatively, we could just make it so all of the `_ENTRIES_PATH_MAP` values are tuples instead of using variadic

which is recommended?